DEMO.md

```diff
@@ -74,6 +74,12 @@ Besides the requirements listed in the [README.md](./README.md) file, we’ll ne
   - [storage-contract](https://github.com/babylonchain/storage-contract).
   - [babylon-timestamping-demo](https://github.com/babylonchain/babylon-timestamping-demo).
 
+### 4. Add Babylon & Vigilante submodules
+  - [Babylon Chain](https://github.com/babylonchain/babylon).
+    - `git submodule add https://github.com/babylonchain/babylon.git`
+  - [Vigilante](https://github.com/babylonchain/vigilante).
+    - `git submodule add https://github.com/babylonchain/vigilante.git`
```

Comment:
This could now be changed to `git submodule init && git submodule update`, as the modules are already added to the repo.